exchanges/poloniex/poloniex_wrapper.go

```diff
@@ -557,10 +557,20 @@ func (p *Poloniex) ModifyOrder(action *order.Modify) (string, error) {
 		action.PostOnly,
 		action.ImmediateOrCancel)
```

Comment:
Similar to the bithumb request, you may want to have checks on these as you  may unintentionally modify an order